homeassistant/components/saj/__init__.py

```diff
@@ -1 +1,24 @@
 """The saj component."""
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+
+PLATFORMS = ["sensor"]
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the saj component."""
+    hass.data[DOMAIN] = {}
```

Comment:
`hass.data[DOMAIN] = {}` is initialised but never used. You can use it to save a instance of `SAJInverter` which then can be used in the platforms.